thirdeye-server/src/main/java/ai/startree/thirdeye/resources/EntityResource.java

```diff
@@ -53,18 +55,23 @@
 public class EntityResource {
 
   private final GenericPojoDao genericPojoDao;
+  private final AuthorizationManager authorizationManager;
 
   @Inject
-  public EntityResource(final GenericPojoDao genericPojoDao) {
+  public EntityResource(final GenericPojoDao genericPojoDao,
```

Comment:
this is another place where I'd prefer `admin` privileges to be able to read internal DB entities.